src/common/utils/index.js

```diff
@@ -10,6 +10,86 @@ export function clamp(value, min, max) {
   return value;
 }
 
+function linearGradient(left, right) {
+  return `linear-gradient(to right, ${left}, ${right})`;
+}
+
+export function mapFromWindowsTheme(name, windowsTheme, useGradients) {
+  /* eslint-disable no-unused-vars */
+  const {
+    ButtonAlternateFace,
+    ButtonDkShadow,
+    ButtonFace,
+    ButtonHilight,
+    ButtonLight,
+    ButtonShadow,
+    ButtonText,
+    ActiveBorder,
+    AppWorkspace,
+    Background,
+    InactiveBorder,
+    Scrollbar,
+    Window,
+    WindowFrame,
+    WindowText,
+    ActiveTitle,
+    GradientActiveTitle,
+    GradientInactiveTitle,
+    InactiveTitle,
+    InactiveTitleText,
+    TitleText,
+    Menu,
+    MenuBar,
+    MenuHilight,
+    MenuText,
+    GrayText,
+    Hilight,
+    HilightText,
+    HotTrackingColor,
+    InfoText,
+    InfoWindow
+  } = windowsTheme;
+  /* eslint-enable no-unused-vars */
+
+  return {
+    name,
+
+    anchor: HotTrackingColor,
+    anchorVisited: GrayText, // no Windows equivalent
```

Comment:
let's map `anchorVisited` to `HotTrackingColor` as well then. This way links will always stay the same color

Reply:
That makes sense. I've made the change.